### DISCUSSION TITLE
[Mia-Platform marketplace] Is possible to set a marketplace element as "singleton" for a project?

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#246] [Ideas]
### Which section?

Marketplace

### Which is the version?

v10.9.0

### Description

Hi guys,
I need to know if there is the possibility to set a marketplace element as a "singleton" for each project.

At the moment every project supports just one single instance of services like:
- crud-service
- api-portal
- swagger-aggregator
- ...

It's possible to set custom services/plugins too as "singleton"?

Thanks.

**Comment by fredmaggiowski** [comment]
Hi, do you some interesting use case in mind for such a feature? I honestly don't see a strong reason to put such a limitation 🤔

**Reply by frengooo** [reply]
We could need it for a PoC about the marketplace we are working on.
Anyway, I suppose is not possible at the moment, right?

**Reply by fredmaggiowski** [reply]
not at the moment, try opening an issue in the internal roadmap so that we can follow up on this later

**Reply by frengooo** [reply]
Opened 😄 @fredmaggiowski now you can work on it 💪

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
